src/seedu/addressbook/commands/ListCommand.java

```diff
@@ -15,11 +15,11 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n" 
             + "Displays all persons in the address book as a list with index numbers.\n\t"
             + "Example: " + COMMAND_WORD;
-
-
-    @Override
```

Comment:
Always good to have `@Override` annotation when you are overriding a method. Think about what happens if you misspell the method you are overriding or when you are overriding a method that does not exists